go.mod

```diff
@@ -5,4 +5,6 @@ go 1.12
 require (
 	github.com/jwalton/gchalk v1.3.0
 	github.com/mattn/go-runewidth v0.0.10
+	github.com/stretchr/testify v1.8.4
```

Comment:
You should no longer need this?

Reply:
You're right. Thanks for pointing this out.